[BUG] Fix flake8 errors

**Describe the bug**
We have a few unwanted changes in the develop branch, which are not according to our flake8 style guide.

**To Reproduce**
Run flake8 according to doc

**Expected behavior**
Successful run of flake8 and Travis ci pull requests

**Screenshots**
`

```
> eurocodedesign/geometry/section.py:1:4: W291 trailing whitespace
> 
> eurocodedesign/geometry/section.py:5:1: F401 'abc.abstractmethod' imported but unused
> 
> eurocodedesign/geometry/steelsections/__init__.py:1:4: W291 trailing whitespace
> 
> eurocodedesign/geometry/steelsections/__init__.py:8:1: F401 'dataclasses.field' imported but unused
> 
> eurocodedesign/geometry/steelsections/__init__.py:127:4: W291 trailing whitespace
> 
> eurocodedesign/geometry/steelsections/__init__.py:133:80: E501 line too long (81 > 79 characters)
> 
> eurocodedesign/geometry/steelsections/__init__.py:134:80: E501 line too long (81 > 79 characters)
> 
> eurocodedesign/geometry/steelsections/__init__.py:135:80: E501 line too long (81 > 79 characters)
> 
> eurocodedesign/geometry/steelsections/__init__.py:136:80: E501 line too long (81 > 79 characters)
> 
> eurocodedesign/geometry/steelsections/__init__.py:177:80: E501 line too long (81 > 79 characters)
> 
> eurocodedesign/geometry/steelsections/__init__.py:191:80: E501 line too long (80 > 79 characters)
> 
> eurocodedesign/geometry/steelsections/__init__.py:221:80: E501 line too long (84 > 79 characters)
> 
> eurocodedesign/geometry/steelsections/__init__.py:239:80: E501 line too long (84 > 79 characters)
> 
> eurocodedesign/geometry/steelsections/__init__.py:260:80: E501 line too long (83 > 79 characters)
> 
> eurocodedesign/geometry/steelsections/__init__.py:266:80: E501 line too long (81 > 79 characters)
> 
> eurocodedesign/geometry/steelsections/__init__.py:270:80: E501 line too long (84 > 79 characters)
> 
> eurocodedesign/geometry/steelsections/__init__.py:288:16: W292 no newline at end of file
> 
> eurocodedesign/materials/structuralsteel.py:1:44: W291 trailing whitespace
> 
> eurocodedesign/materials/structuralsteel.py:3:78: W291 trailing whitespace
> 
> eurocodedesign/materials/structuralsteel.py:13:1: W293 blank line contains whitespace
> 
> eurocodedesign/materials/structuralsteel.py:19:80: E501 line too long (87 > 79 characters)
> 
> eurocodedesign/materials/structuralsteel.py:20:80: E501 line too long (88 > 79 characters)
> 
> eurocodedesign/materials/structuralsteel.py:21:1: W293 blank line contains whitespace
> 
> eurocodedesign/materials/structuralsteel.py:24:1: W293 blank line contains whitespace
> 
> eurocodedesign/materials/structuralsteel.py:25:39: W291 trailing whitespace
> 
> eurocodedesign/materials/structuralsteel.py:27:47: W291 trailing whitespace
> 
> eurocodedesign/materials/structuralsteel.py:108:4: W291 trailing whitespace
> 
> eurocodedesign/materials/structuralsteel.py:114:4: W291 trailing whitespace
> 
> eurocodedesign/materials/structuralsteel.py:122:1: W293 blank line contains whitespace
> 
> tests/test_eurocodedesign.py:8:1: F401 'eurocodedesign.eurocodedesign' imported but unused
> 
> tests/test_steelsections.py:1:4: W291 trailing whitespace
> 
> tests/test_steelsections.py:62:5: F841 local variable 'index' is assigned to but never used
> 
> tests/test_steelsections.py:140:40: E712 comparison to True should be 'if cond is True:' or 'if cond:'
> 
> tests/test_steelsections.py:144:34: E712 comparison to False should be 'if cond is False:' or 'if not cond:'
> 
> tests/test_steelsections.py:148:40: E712 comparison to False should be 'if cond is False:' or 'if not cond:'
> 
> tests/test_steelsections.py:164:12: F632 use ==/!= to compare constant literals (str, bytes, int, float, tuple)
> 
> tests/test_steelsections.py:168:39: F841 local variable 'error_info' is assigned to but never used
> 
> tests/test_steelsections.py:169:6: E111 indentation is not a multiple of 4
> 
> tests/test_steelsections.py:175:10: F841 local variable 'error_info' is assigned to but never used
> 
> tests/test_steelsections.py:176:6: E111 indentation is not a multiple of 4
> 
> tests/test_steelsections.py:182:10: F841 local variable 'error_info' is assigned to but never used
> 
> tests/test_steelsections.py:183:6: E111 indentation is not a multiple of 4
> 
> tests/test_steelsections.py:195:1: E302 expected 2 blank lines, found 1
> 
> tests/test_steelsections.py:198:37: E225 missing whitespace around operator
> 
> tests/test_structuralsteel.py:47:35: E251 unexpected spaces around keyword / parameter equals
```

`
.

--- a/eurocodedesign/geometry/section.py
+++ b/eurocodedesign/geometry/section.py
@@ -1,8 +1,7 @@
-""" 
+"""
 classes for sections of various materials
 """
-
-from abc import ABC, abstractmethod
+from abc import ABC
 from dataclasses import dataclass, field
 
 

--- a/eurocodedesign/geometry/steelsections/__init__.py
+++ b/eurocodedesign/geometry/steelsections/__init__.py
@@ -1,11 +1,11 @@
-""" 
+"""
 STEEL PROFILE CLASSES
 """
 import os
 import pandas as pd
 
 from pathlib import Path
-from dataclasses import dataclass, field
+from dataclasses import dataclass
 from eurocodedesign.geometry.section import BasicSection
 
 
@@ -124,18 +124,24 @@ class SquareHollowSection(HollowSection):
     manufacture_method: str
 
 
-""" 
+"""
 MODULE LEVEL CONSTANTS
 """
 
 
 _SECTION_DATA = {
-    "HEA": {"filename": "hea_en10365_2017.csv", "section_class": RolledISection},
-    "HEB": {"filename": "heb_en10365_2017.csv", "section_class": RolledISection},
-    "HEM": {"filename": "hem_en10365_2017.csv", "section_class": RolledISection},
-    "IPE": {"filename": "ipe_en10365_2017.csv", "section_class": RolledISection},
-    "UB": {"filename": "ub_en10365_2017.csv", "section_class": RolledISection},
-    "UC": {"filename": "uc_en10365_2017.csv", "section_class": RolledISection},
+    "HEA": {"filename": "hea_en10365_2017.csv",
+            "section_class": RolledISection},
+    "HEB": {"filename": "heb_en10365_2017.csv",
+            "section_class": RolledISection},
+    "HEM": {"filename": "hem_en10365_2017.csv",
+            "section_class": RolledISection},
+    "IPE": {"filename": "ipe_en10365_2017.csv",
+            "section_class": RolledISection},
+    "UB": {"filename": "ub_en10365_2017.csv",
+           "section_class": RolledISection},
+    "UC": {"filename": "uc_en10365_2017.csv",
+           "section_class": RolledISection},
     "CHS": {
         "filename": "chs_en10219_en10210_2006.csv",
         "section_class": CircularHollowSection,
@@ -174,7 +180,8 @@ def _is_valid_type(section_name: str) -> bool:
     """
     for section_type in _SECTION_DATA.keys():
         if section_type in section_name:
-            filepath = _get_data_path() / _SECTION_DATA[section_type]["filename"]
+            filepath = _get_data_path() \
+                       / _SECTION_DATA[section_type]["filename"]
             return os.path.exists(filepath)
     return False
 
@@ -188,8 +195,8 @@ def _import_section_database(section_type: str) -> pd.DataFrame:
         section_type (str): type of section e.g. "IPE", "CHS", "UB", etc.
 
     Returns:
-        pd.DataFrame: containing all the geometric data for the profiles for the
-            provided section_type
+        pd.DataFrame: containing all the geometric data for
+        the profiles for the provided section_type
     """
     filepath = _get_data_path() / _SECTION_DATA[section_type]["filename"]
     return pd.read_csv(filepath, index_col=0)
@@ -218,7 +225,8 @@ def _get_section_type(section_name: str) -> str | None:
     until the occurence of the first digit.
 
     Args:
-        section_name (str): the name of the steel section e.g. "IPE100", "CHS63x2.3"
+        section_name (str): the name of the steel section
+                            e.g. "IPE100", "CHS63x2.3"
 
     Returns:
         str|None: the first letters of the name. None if there are no digits in
@@ -236,7 +244,8 @@ def _load_section_props(section_name: str) -> pd.Series:
     """retrieves the section properties for the given section
 
     Args:
-        section_name (str): the name of the steel section e.g. "IPE100", "CHS63x2.3"
+        section_name (str): the name of the steel section
+                            e.g. "IPE100", "CHS63x2.3"
 
     Raises:
         ValueError: occurs when input is not a string
@@ -257,17 +266,19 @@ def _load_section_props(section_name: str) -> pd.Series:
             else:
                 raise ValueError(f"Invalid section name: '{section_name}'")
         else:
-            raise ValueError(f"Invalid section type for section: '{section_name}'")
+            raise ValueError(f"Invalid section type for section: "
+                             f"'{section_name}'")
 
 
 def _get_section(section_name: str) -> SteelSection:
     """gets the geometric properties of the section
 
-    creates and SteelSection object with the appropriate geometric properties for
-    the section with the name section_name
+    creates and SteelSection object with the appropriate geometric
+     properties for the section with the name section_name
 
     Args:
-        section_name (str): the name of the steel section e.g. "IPE100", "CHS63x2.3"
+        section_name (str): the name of the steel section
+                            e.g. "IPE100", "CHS63x2.3"
 
     Returns:
         SteelSection: object containing the geometric properties of the section

--- a/eurocodedesign/materials/structuralsteel.py
+++ b/eurocodedesign/materials/structuralsteel.py
@@ -1,6 +1,6 @@
-""" Material properties of structural steel 
+""" Material properties of structural steel
 
-Classes defining the different types of construction steel that are permitted 
+Classes defining the different types of construction steel that are permitted
 within the Eurocode framework. The different steel types are defined by the
 following standards:
     -- EN 10025-2
@@ -10,21 +10,23 @@ following standards:
     -- EN 10025-6
     -- EN 10210-1
     -- EN 10219-1
-    
+
 All material strengths are given in MPa (N/mm2) unless otherwise indicated
 
 Example Usage:
 
     import eurocodedesign.materials.structuralsteel as ss
-    steel_material = ss.get("S235", True) # S235 steel material with thickness <= 40 mm
-    steel_material = ss.get("S355", False) # S355 steel material with thickness >= 40 mm
-    
+    # S235 steel material with thickness <= 40 mm
+    steel_material = ss.get("S235", True)
+    # S355 steel material with thickness >= 40 mm
+    steel_material = ss.get("S355", False)
+
     yield_stress = steel_material.f_yk
     ultimate_stress = steel_material.f_uk
-    
-    elastic_modulus = steel_material.E 
+
+    elastic_modulus = steel_material.E
     shear_modulus = steel_material.G
-    thermal_coefficient = steel_material.alpha  
+    thermal_coefficient = steel_material.alpha
 """
 
 from dataclasses import dataclass, field
@@ -105,13 +107,13 @@ class S450(BasicStructuralSteel):
     norm: str = field(default="EN 10025-2", kw_only=True)
 
 
-""" 
+"""
 Constants
 """
 
 STEEL_TYPES = {"S235": S235, "S275": S275, "S355": S355, "S450": S450}
 
-""" 
+"""
 Definitions
 """
 
@@ -119,7 +121,7 @@ Definitions
 def get(
     steel_type: str, thickness_less_than_equal_40mm: bool = True
 ) -> BasicStructuralSteel:
-    
+
     if steel_type in STEEL_TYPES.keys():
         return STEEL_TYPES[steel_type](thickness_less_than_equal_40mm)
     else:

--- a/tests/test_eurocodedesign.py
+++ b/tests/test_eurocodedesign.py
@@ -5,7 +5,7 @@
 import pytest
 
 
-from eurocodedesign import eurocodedesign
+from eurocodedesign import eurocodedesign as ed # noqa
 
 
 @pytest.fixture

--- a/tests/test_steelsections.py
+++ b/tests/test_steelsections.py
@@ -1,4 +1,4 @@
-""" 
+"""
 tests for the module responsible for loading steel section geometry
 eurocodedesign.geometry.steelsection.manager
 """
@@ -59,7 +59,6 @@ def dummy_IPE270():
 @pytest.fixture
 def IPE_dataframe():
     # set up test section database for the
-    index = ["IPE100", "IPE270"]
     geometric_properties_IPE270 = [
         270,
         135,
@@ -137,15 +136,15 @@ def IPE_dataframe():
 
 
 def test_when_is_valid_type():
-    assert ss._is_valid_type("HEM600") == True
+    assert ss._is_valid_type("HEM600") is True
 
 
 def test_when_not_is_valid_type_with_empty_string():
-    assert ss._is_valid_type("") == False
+    assert ss._is_valid_type("") is False
 
 
 def test_when_not_is_valid_type_with_invalid_name():
-    assert ss._is_valid_type("XRX120") == False
+    assert ss._is_valid_type("XRX120") is False
 
 
 def test_when_is_valid_section(section_dataframe):
@@ -161,26 +160,26 @@ def test_when_get_section_type_is_found():
 
 
 def test_when_get_section_type_is_not_found():
-    assert ss._get_section_type("320LRB") is ""
+    assert ss._get_section_type("320LRB") == ""
 
 
 def test_load_section_props_input_not_string():
-    with pytest.raises(ValueError) as error_info:
-     ss._load_section_props(2)
+    with pytest.raises(ValueError):
+        ss._load_section_props(2)
 
 
 def test_load_section_props_input_is_wrong_type():
     with pytest.raises(
         ValueError, match="Invalid section type for section: 'XYZ281'"
-    ) as error_info:
-     ss._load_section_props("XYZ281")
+    ):
+        ss._load_section_props("XYZ281")
 
 
 def test_load_section_props_input_is_wrong_section():
     with pytest.raises(
         ValueError, match="Invalid section name: 'IPE281'"
-    ) as error_info:
-     ss._load_section_props("IPE281")
+    ):
+        ss._load_section_props("IPE281")
 
 
 @patch("eurocodedesign.geometry.steelsections._import_section_database")
@@ -192,7 +191,8 @@ def test_load_section_props_for_valid_section_name(
     expected = profile_series
     assert all(actual == expected) is True
 
+
 @patch("eurocodedesign.geometry.steelsections._import_section_database")
 def test_get_section(section_data, IPE_dataframe, dummy_IPE270):
     section_data.return_value = IPE_dataframe
-    assert ss._get_section("IPE270")==dummy_IPE270
+    assert ss._get_section("IPE270") == dummy_IPE270

--- a/tests/test_structuralsteel.py
+++ b/tests/test_structuralsteel.py
@@ -44,7 +44,7 @@ def test_get_for_structural_steel_types_with_steel_in_library_t_greater_40(
 
 
 def test_get_for_structural_steel_types_with_steel_not_in_library():
-    with raises(ValueError, match= "Steel material '211' not in library"):
+    with raises(ValueError, match="Steel material '211' not in library"):
         ss.get("211", True)
 
 

